### DISCUSSION TITLE
Disabling fly on join

### DISCUSSION THREAD

**Discussion by luketeam5** [EssentialsX/Essentials#4910] [Q&A] [answered]
Hello,
I would like to disable the feature that automatically gives admins /fly when they connect to the server (if they disconnected while flying with elytra, in the air, ...).
I tried to set `send-fly-enable-on-join: true` to false however this only turns off the notification, not /fly.
![image](https://user-images.githubusercontent.com/40070133/169668491-ed14899d-b7d4-447c-92e1-8d9eef13f6b3.png)

Thanks for any help.

**Comment by osfanbuff63** [comment, marked as answer]
This can be turned off by negating (setting to false) the permission `essentials.fly.safelogin` for the admin group in your permissions plugin.
